### ISSUE TITLE
After upgrading anaconda 4.3.29 -> 4.4.10-py27_0 "source activate" no longer works

### ISSUE THREAD

**Issue by durack1** [ContinuumIO/anaconda-issues#8540]
An example of the issue is below.

Valid environments exist:
```
duro@ocean:[~]:[9590]> conda env list
# conda environments:
#
base                  *  /export/duro/anaconda2
2.12                     /export/duro/anaconda2/envs/2.12
2.12-nox                 /export/duro/anaconda2/envs/2.12-nox
cdat212cmor328Nightlynclnco     /export/duro/anaconda2/envs/cdat212cmor328Nightlynclnco
cmor328                  /export/duro/anaconda2/envs/cmor328
cmor330                  /export/duro/anaconda2/envs/cmor330
cmor330_1                /export/duro/anaconda2/envs/cmor330_1
cmorNightly180201        /export/duro/anaconda2/envs/cmorNightly180201
cmorNightly180213        /export/duro/anaconda2/envs/cmorNightly180213
pmp-1.1.2                /export/duro/anaconda2/envs/pmp-1.1.2
pmp-nightly171121        /export/duro/anaconda2/envs/pmp-nightly171121
pyferret7p1              /export/duro/anaconda2/envs/pyferret7p1
uvcdat2100               /export/duro/anaconda2/envs/uvcdat2100
uvcdat2100nox            /export/duro/anaconda2/envs/uvcdat2100nox
uvcdat2120               /export/duro/anaconda2/envs/uvcdat2120
uvcdat280                /export/duro/anaconda2/envs/uvcdat280
uvcdat280nox             /export/duro/anaconda2/envs/uvcdat280nox
uvcdatNightly180126      /export/duro/anaconda2/envs/uvcdatNightly180126
                         /usr/local/anaconda2/envs/duroenv
                         /usr/local/anaconda2/envs/duroenv-robust

```
However, I am unable to load them using the `source activate` call:
```
duro@ocean:[~]:[9590]> source activate cmorNightly180213
/export/duro/anaconda2/etc/profile.d/conda.sh
bash: _conda_activate: command not found
```

**Comment by durack1** [comment]
This is possibly a dupe of conda/conda#6667

**Comment by durack1** [comment]
It will be interesting to see if conda/conda#6669 (once merged, and a new conda is released) will fix this issue for me

**Comment by durack1** [comment]
@nehaljwani it seems that the steps outlined in conda/conda#6667 works, which if I'm reading this correctly will be fixed when the PR conda/conda#6669 is merged

```
duro@ocean:[~]:[9590]> source activate cmorNightly180213
/export/duro/anaconda2/etc/profile.d/conda.sh
bash: _conda_activate: command not found
...
duro@ocean:[~]:[9592]> unalias .
duro@ocean:[~]:[9593]> source activate cmorNightly180213
(cmorNightly180213) duro@ocean:[~]:[9592]>
```

As there is an open issue for this (conda/conda#6667) I'll close this dupe

**Comment by nehaljwani** [comment]
Thank you for updating the issue.
